soc/arm/st_stm32/stm32l0/dts_fixup.h

```diff
@@ -88,4 +88,11 @@
 
 #define DT_ADC_1_NAME			DT_ST_STM32_ADC_40012400_LABEL
 
+#define DT_RTC_0_BASE_ADDRESS           DT_ST_STM32_RTC_40002800_BASE_ADDRESS
+#define DT_RTC_0_IRQ_PRI                DT_ST_STM32_RTC_40002800_IRQ_0_PRIORITY
+#define DT_RTC_0_IRQ                    DT_ST_STM32_RTC_40002800_IRQ_0
+#define DT_RTC_0_NAME                   DT_ST_STM32_RTC_40002800_LABEL
+#define DT_RTC_0_CLOCK_BITS             DT_ST_STM32_RTC_40002800_CLOCK_BITS
+#define DT_RTC_0_CLOCK_BUS              DT_ST_STM32_RTC_40002800_CLOCK_BUS
```

Comment:
These should not be needed anymore after rebase.

Reply:
They are for the sample application:
```
.../zephyr/samples/drivers/counter/alarm/src/main.c: In function 'main':
.../zephyr/samples/drivers/counter/alarm/src/main.c:61:35: error: 'DT_RTC_0_NAME' undeclared (first use in this function); did you mean 'DT_I2C_2_NAME'?
   61 |  counter_dev = device_get_binding(DT_RTC_0_NAME);
      |                                   ^~~~~~~~~~~~~
      |                                   DT_I2C_2_NAME
.../zephyr/samples/drivers/counter/alarm/src/main.c:61:35: note: each undeclared identifier is reported only once for each function it appears in
[34/113] Building C object zephyr/CMakeFiles/zephyr.dir/drivers/timer/cortex_m_systick.c.obj
ninja: build stopped: subcommand failed.
FATAL ERROR: command exited with status 1: cmake --build /..../zephyr/build
serafin@sl-baw:../zephyr$ cd build/
serafin@sl-baw:../zephyr/build$ grep -nrH RTC_0
zephyr/b_l072z_lrwan1.dts.pre.tmp:49:   label = "RTC_0";
zephyr/include/generated/devicetree.conf:703:DT_ST_STM32_RTC_40002800_LABEL="RTC_0"
zephyr/include/generated/devicetree.conf:704:DT_INST_0_ST_STM32_RTC_LABEL="RTC_0"
zephyr/include/generated/devicetree_unfixed.h:772:#define DT_ST_STM32_RTC_40002800_LABEL              "RTC_0"
zephyr/zephyr.dts:47:			label = "RTC_0";
```

Reply:
My bad, I was not precise enough: Please remove all except `DT_RTC_0_NAME`.